app/view/account_type.py

```diff
@@ -0,0 +1,107 @@
+                                     status=status.HTTP_201_CREATED)
+        return response.success_response()
+
+    def get(self):
+
+        """Provides the data of all the accountTypes"""
+
+        all_account_types = AccountType.query.all()
+        output = []
+        for account in all_account_types:
+            currentaccount = {}
+            currentaccount['id'] = account.id
+            currentaccount['account_type'] = account.account_type
+            output.append(currentaccount)
+        logger.info("All account_type data returned successfully")
+        response = ResponseGenerator(data=output, message="All account_type data returned successfully",
+                                     success=True, status=status.HTTP_200_OK)
+        return response.success_response()
+
+
+class AccountTypeData(Resource):
+
+    """AccountTypeData for GET(single user), PUT(update user), DELETE(delete user)"""
+
+    def get(self, id):
+
+        """Gives the data of single account_type with selected account_type_id """
+
+        account = AccountType.query.filter(AccountType.id == id).first()
+        output = account_type_schema.dump(account)
+        logger.info('AccountType data returned successfully')
+        if account:
+            response = ResponseGenerator(data=output, message="AccountType data returned successfully", success=True,
+                                         status=status.HTTP_200_OK)
+            return response.success_response()
+        else:
+            logger.exception("AccountType id not found")
+            response = ResponseGenerator(data={}, message="AccountType id not found", success=False,
+                                         status=status.HTTP_404_NOT_FOUND)
+            return response.error_response()
+
+    def put(self, id):
+
+        """Update the account type data """
+
+        data = request.get_json()
+        result = account_type_schema.validate(data)
+        if result:
+            logger.warning('Missing or sending incorrect data to update an activity')
+            response = ResponseGenerator(data={}, message="Missing or sending incorrect data to update an activity",
+                                         success=False, status=status.HTTP_400_BAD_REQUEST)
+            return response.error_response()
+        account = AccountType.query.filter(AccountType.id == id).first()
+        if account:
+            account.id = data.get('id', account.id)
+            account.account_type = data.get('account_type', account.account_type)
+            db.session.commit()
+            output = account_type_schema.dump(account)
+            logger.info("AccountType data updated successfully")
+            response = ResponseGenerator(data=output, message="AccountType data updated successfully", success=True,
+                                         status=status.HTTP_200_OK)
+            return response.success_response()
```

Comment:
You will update the account if the account with that id is present. What if account with given id is not present? You should send error response with appropriate message.